homeyess/website/views.py

```diff
@@ -392,6 +411,28 @@ def confirm_ride(request, ride_id):
 
     return redirect('dashboard')
 
+def filterQuerySetJobs(jobs, location, job_title):
+    # If no filter has been applied, don't filter:
+    if location is None and job_title is None:
+        return jobs
+
+    # This way we can filter if only one of the values is set:
+    loc_filter = location.lower() if location is not None else ''
+    title_filter = job_title.lower() if job_title is not None else ''
+
+    return list(filter(lambda job: (loc_filter in job.location.lower()) and (title_filter in job.job_title.lower()), jobs))
+
+def filterQuerySetJobs(jobs, location, job_title):
+    # If no filter has been applied, don't filter:
+    if location is None and job_title is None:
+        return jobs
+
+    # This way we can filter if only one of the values is set:
+    loc_filter = location.lower() if location is not None else ''
+    title_filter = job_title.lower() if job_title is not None else ''
+
+    return list(filter(lambda job: (loc_filter in job.location.lower()) and (title_filter in job.job_title.lower()), jobs))
+
```

Comment:
please put this in a file `job_utils.py` in the same directory. I am doing the same for ride util functions